guides/source/routing.md

```diff
@@ -1359,6 +1410,8 @@ For example, here's a small section of the `bin/rails routes` output for a RESTf
 edit_user GET    /users/:id/edit(.:format) users#edit
 ```
 
+The route name (`new_user` above for example) can be considered the base for deriving route helpers. For example to get the name of a route helper, add the suffix “_path” or “_url” to the route name(`new_user_path` for example).
```

Comment:
[The rendered guides](https://ce08d9e7.rails-docs-preview.pages.dev/guides/routing#inspecting-routes) is recognizing the underscores as italics. I'd say we should use a code block here:

```suggestion
The route name (`new_user` above, for example) can be considered the base for deriving route helpers. To get the name of a route helper, add the suffix `_path` or `_url` to the route name(`new_user_path`, for example).
```